File: day1.py
f = open("day1.input")
line = f.readline()
total = 0
max = 0
while line:
    line = line.strip("\n")
    if line=='':
        max = max if max>total else total
        total = 0
    else:
        line = int(line)
        total+=line
    line = f.readline()

print(max)

File: day1.php
<?php


$fp = fopen("input.txt", "r");
$total = 0;
$max =0;

while(!feof($fp)){
    $line = trim(fgets($fp));

    if(empty($line)){
       $max = $max > $total ? $max : $total; 
       $total = 0;
}else{
    $total += intval($line);
}
}

var_dump($max); 
fclose($fp);



